about.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/c3c0d932d2.js" crossorigin="anonymous"></script>
+  <title>About</title>
+</head>
+<body>
+  <header class="navbar">
+    <i class="fa-solid fa-bars menu-btn" id="menu-btn"></i>
+    <div class="navbar-menu">
+      <div class="menu-options">
+        <ul class="options">
+          <li class="nav-icon"><i class="fa-brands fa-facebook"></i></li>
+          <li class="nav-icon"><i class="fa-brands fa-square-twitter"></i></li>
+          <li class="nav-option">English</li>
+          <li class="nav-option">My page</li>
+          <li class="nav-option">Logout</li>
+        </ul>
+      </div>
+      <nav class="menu-links" id="menu">
+        <i class="fa-regular fa-circle-xmark close-btn menu-btn" id="close-btn"></i>
+        <a href="/home.html"><img src='media/event-logo.png' alt='logo' class='nav-logo' id='nav-logo'></a>
+        <ul class="nav-links">
+          <li class="nav-link"><a class='about-link' href="#" id="about-link">About</a></li>
+          <li class="nav-link"><a href="#">Program</a></li>
+          <li class="nav-link"><a href="#">Join</a></li>
+          <li class="nav-link"><a href="#">Sponsor</a></li>
+          <li class="nav-link"><a href="#">News</a></li>
+          <li class="nav-link"><a href="#">YWCC Campaign</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+
+  <section class="about">
+    <h1 class="p-title">YEARLY WORLDWIDE COMEDIANS CONFERENCE</h1>
+    <p class="about-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quae porro impedit quod omnis repellendus amet consequatur iste aspernatur dolorem! Quaerat obcaecati excepturi recusandae molestias numquam.</p>
+    <div class="contact">
+      <p class="contact-description">Please contact us per Email for any further questions about YW Comedians Conference 2022</p>
+      <p class="email">ywcc2022@ywspain.com</p>
+    </div>
+  </section>
```

Comment:
You're almost there, but you still have some opportunities to improve by following the sizing from the original design idea. 

Here I'd highly recommend increasing the text size.

Yours
![image](https://user-images.githubusercontent.com/34779928/203858316-f525e987-7196-45cb-9f5a-ec77871a01fb.png)

Original
![image](https://user-images.githubusercontent.com/34779928/203858351-abdf7615-6cd9-493c-8d09-074f6d2903cd.png)